Emphasize eval results metadata more (#632)
huggingface/huggingface_hub: docs/hub/model-repos.md

* License identifiers are the keywords listed in the right column of [this table](#list-of-license-identifiers).
* Dataset, metric, and language identifiers are those listed on the [Datasets](https://huggingface.co/datasets), [Metrics](https://huggingface.co/metrics) and [Languages](https://huggingface.co/languages) pages and in the [`datasets`](https://github.com/huggingface/datasets) repository.

> You can even specify your **model's eval results** in a structured way, which will allow the Hub to parse, display, and even link them to Papers With Code leaderboards. Specify this information [here](https://github.com/huggingface/huggingface_hub/blame/main/modelcard.md).

Author: ```suggestion
You can even specify your **model's eval results** in a structured way, which will allow the Hub to parse, display, and even link them to Papers With Code leaderboards. See how to format this data [in the metadata spec](https://github.com/huggingface/huggingface_hub/blame/main/modelcard.md).
```

Author: changed the wording for this @stevhliu because what i mean is that the linked document *is* the spec

Reviewer: Ah ok, my bad! For some reason I thought the link was to a model card `README.md` file on the Hub.

> You can even specify your **model's eval results** in a structured way, which will allow the Hub to parse them, display them, and even link to Papers With Code leaderboards. Again the specification for this is [here](https://github.com/huggingface/huggingface_hub/blame/main/modelcard.md).

Reviewer: ```suggestion
You can even specify your **model's eval results** in a structured way, which will allow the Hub to parse, display, and even link them to Papers With Code leaderboards. Specify this information [here](https://github.com/huggingface/huggingface_hub/blame/main/modelcard.md).
```
